source/macos/AdaptiveCards/AdaptiveCards/Managers/ACSVisibilityManager.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ACSVisibilityManager.swift
+//  AdaptiveCards
+//
+//  Created by uchauhan on 30/08/22.
+//
+//  This class is designed to handle the visibility of views as well as the corresponding padding or separator. Class also depended on Filler Space Manager to find associated views.
+
+import AdaptiveCards_bridge
+import AppKit
+
+/// VisibilityManager protocols for handle element visibility
+@objc protocol ACSIVisibilityManagerFacade {
+    func hideView(_ view: NSView)
+    func unhideView(_ view: NSView)
+}
```

Comment:
Why is there an I in ACSIVisibilityFacade?

Reply:
General form -
ACS = Our general prefix 
I = Implementations
Facade = it's perform and handle for many input from different place. front side wrapper.